CHANGELOG.md

```diff
@@ -24,18 +24,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Config option `cursor.blink_timeout` to timeout cursor blinking after inactivity
 - Escape sequence to set hyperlinks (`OSC 8 ; params ; URI ST`)
 - Config `hints.enabled.hyperlinks` for hyperlink escape sequence hint highlight
+- On macOS, Alacritty now reads `AppleFontSmoothing` from user defaults to control font smoothing
+- A `removed` config attribute, to show warnings for config fields that have been removed
```

Comment:
This doesn't make any sense for the end users, so could remove it.

Reply:
Sure.